js/tests/karma.conf.js

```diff
@@ -106,7 +106,8 @@ if (browserStack) {
   conf.browsers = browsersKeys
   reporters.push('BrowserStack', 'kjhtml')
   files = files.concat([
-    { pattern: 'js/src/**/*.spec.js', watched: false }
+    { pattern: 'js/src/**/*.spec.js', watched: false },
+    { pattern: 'dist/css/bootstrap.css', watched: false, included: false, served: true }
```

Comment:
please do not include the whole Bootstrap css, just had what's needed to be include